src/gsi/xtofca_wrf.F90

```diff
@@ -0,0 +1,145 @@
+#ifdef FCA_REF_MOD
```

Comment:
Clean up unused comments, add proper documentation in the head.

Reply:
ok, comments cleaned up